### ISSUE TITLE
Grayscale album arts showing monet colors

### ISSUE THREAD

**Issue by itsmeteddyhere** [Malopieds/InnerTune#117]
### Checklist

- [X] I am able to reproduce the bug with the [latest debug version](https://github.com/z-huang/InnerTune/actions).
- [X] I've checked that there is no open or closed issue about this bug.
- [X] This issue contains only one bug.
- [X] The title of this issue accurately describes the bug.

### Steps to reproduce the bug

Play a song that has a grayscale album art (or the main color is a grayscale color)

### Expected behavior

Make the app theme gray

### Actual behavior

Makes the app theme the monet theme that I have set on my phone

### Screenshots/Screen recordings

![Screenshot_20240609-225629_InnerTune](https://github.com/Malopieds/InnerTune/assets/85173021/d154d197-3537-41a6-8aff-57b3af987957)

### Logs

I don't think this is needed in this case

### InnerTune version

v0.5.7-rc1

### Android version

Android 14

**Comment by Malopieds** [comment]
Looks like it takes the color scheme of the previous thumbnail. I want to change the color for some time but I don't have any idea how.

**Comment by itsmeteddyhere** [comment]
for me no matter what song was before it just ises my accent color that for example my themed icons use, just the monet color

**Comment by Malopieds** [comment]
> for me no matter what song was before it just ises my accent color that for example my themed icons use, just the monet color

Yep it's the same color as the monet, but because from black and white it just can't create a palette of colors, what would be an actual solution here ?

**Comment by itsmeteddyhere** [comment]
I would say just make it gray, android can do it (it does it in my music player notification and really anything else) so why not do it in InnerTune

**Comment by Malopieds** [comment]
It should not fail to grab colors anymore, could you check the latest debug version please?

**Comment by itsmeteddyhere** [comment]
seems to be the same 
![Screenshot_20240819-213649](https://github.com/user-attachments/assets/f2eb2053-4caf-4806-bb5c-2b1fe22b7abf)

**Comment by Malopieds** [comment]
It's not the same colors?

**Comment by itsmeteddyhere** [comment]
it's still the Monet colors, not the album art colors
